File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/smart_pointer_rename_runme.m
# do not dump Octave core
if exist("crash_dumps_octave_core", "builtin")
  crash_dumps_octave_core(0);
endif

smart_pointer_rename

f = Foo();
b = Bar(f);

if (b.test() != 3)
    error
endif

if (b.ftest1(1) != 1)
    error
endif

if (b.ftest2(2,3) != 2)
    error
endif

File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/smart_pointer_rename_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'smart_pointer_rename'

include Smart_pointer_rename

f = Foo.new
b = Bar.new(f)

raise RuntimeError if b.test() != 3

raise RuntimeError if b.ftest1(1) != 1

raise RuntimeError if b.ftest2(2,3) != 2
